easybuild/easyconfigs/s/setuptools/setuptools-41.0.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "PythonPackage"
+
+name = "setuptools"
+version = "41.0.1"
+
+homepage = "https://pypi.org/project/setuptools"
+description = """Easily download, build, install, upgrade, and uninstall Python packages"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_ZIP]
+checksums = ['a222d126f5471598053c9a77f4b5d4f26eaa1f150ad6e01dcf1a42e185d05613']
+
+download_dep_fail = True
+use_pip = False
```

Comment:
I'm not sure installing `setuptools` without `pip` is actually still supported (but setting it to `True` may cause problems too, since we don't have any logic in place to favor `pip3` over `pip`)...

Reply:
Actually, I changed also the EasyBlock used to `PythonPackage` and set those after your recommendation for the ReFrame easyconfig, cos otherwise the CI was failing.